src/components/common/TmDataError.vue

```diff
@@ -6,7 +6,7 @@
     <div slot="subtitle">
       Even though you're connected a full node, we can't display this data for
       you right now. Please try again later or
-      <a href="https://github.com/cosmos/voyager/issues">file a bug report</a>.
+      <a href="https://github.com/luniehq/lunie/issues">file a bug report</a>.
```

Comment:
fabo proposed that we change this to open the intercom help box. if you feel like sneaking it in here that would be ok with me 😉

Reply:
Yep, good idea to show that option instead the cold link to the issues page :-)

Reply:
let's do in a different PR